code/statsd/src/main/scala/com/avast/datadog4s/statsd/event/EventerImpl.scala

```diff
@@ -0,0 +1,45 @@
+package com.avast.datadog4s.statsd.event
+
+import cats.effect.Sync
+import com.avast.datadog4s.api.Tag
+import com.avast.datadog4s.api.event.{ Eventer, _ }
+import com.timgroup.statsd.{ StatsDClient, Event => SEvent }
+
+import scala.collection.immutable.Seq
+class EventerImpl[F[_]: Sync](statsDClient: StatsDClient, defaultTags: Seq[Tag]) extends Eventer[F] {
+  private[this] val F = Sync[F]
+
+  private val toPriority: Priority.Value => SEvent.Priority = {
```

Comment:
Use `private[this]` please. It forces compiler to generate simple class attribute.

Reply:
Will do! Learned something new 🙂